packages/nextjs/package.json

```diff
@@ -63,21 +63,20 @@
     "docgen": "pnpm typedoc && docgen --config ./nextjs.docgen.json"
   },
   "dependencies": {
-    "@nhost/nhost-js": "workspace:*",
+    "@nhost/react": "workspace:*",
     "cross-fetch": "^3.1.5",
     "js-cookie": "^3.0.1",
     "xstate": "^4.33.5"
   },
   "peerDependencies": {
-    "@nhost/react": "workspace:*",
     "next": "^12.0.10 || ^13.0.0",
     "react": "^17.0.0 || ^18.0.0",
     "react-dom": "^17.0.0 || ^18.0.0"
   },
   "devDependencies": {
+    "@nhost/nhost-js": "workspace:*",
```

Comment:
Why do we need these as dev dependencies btw? These are all re-exported by @nhost/nhost-js, aren't they?

Reply:
Yes, sorry. I was looking at another package.json while writing the comment.